Core/MIPS/x86/X64IRCompALU.cpp

```diff
@@ -261,10 +308,54 @@ void X64JitBackend::CompIR_HiLo(IRInst inst) {
 
 	switch (inst.op) {
 	case IROp::MtLo:
+#if PPSSPP_ARCH(AMD64)
+		regs_.MapWithExtra(inst, { { 'G', IRREG_LO, 2, MIPSMap::DIRTY } });
+		// First, clear the bits we're replacing.
+		MOV(64, R(SCRATCH1), Imm64(0xFFFFFFFF00000000ULL));
+		AND(64, regs_.R(IRREG_LO), R(SCRATCH1));
```

Comment:
I wonder if this is better than shifting into a wall and back? The latter is certainly smaller in bytes.

Reply:
Yeah, I'm not really sure.  I imagine this is pretty fast, but yes the shifts would be about 2 bytes smaller I think?

-[Unknown]

Reply:
That 64-bit MOV alone is 10 bytes I think, so 12 or 13? And shifts would be 6 bytes in total. Or I suppose 8 since rex bytes are needed...

So yeah, not a huge difference in the big pictures. 6-7 bytes saved..

Reply:
Oh right, I forgot the AND, which needs the REX as well.  I don't think MtLo is very common though, maybe I should just switch to shifts... probably won't impact much anyway.

-[Unknown]